hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/table/HoodieTableSource.java

```diff
@@ -396,6 +396,8 @@ private List<MergeOnReadInputSplit> buildInputSplits() {
             .rowType(this.tableRowType)
             .maxCompactionMemoryInBytes(maxCompactionMemoryInBytes)
             .partitionPruner(partitionPruner)
+            .skipCompaction(conf.getBoolean(FlinkOptions.READ_STREAMING_SKIP_COMPACT))
```

Comment:
The `read.streaming.skip_compaction` and `read.streaming.skip_clustering` is the option of streaming read, not batch read.

Reply:
> The `read.streaming.skip_compaction` and `read.streaming.skip_clustering` is the option of streaming read, not batch read.

We can add a similar batch configuration.